docs/docsite/rst/shared_snippets/basic_concepts.txt

```diff
@@ -1,34 +1,56 @@
 
-Collections
-===========
+Playbooks
+=========
+
+They contain Plays (which are the basic unit of Ansible exeuction), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates on.
+Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
+
+
+Plays
+-----
+
+The playbook object that maps manged nodes (hosts) to tasks, all Ansible execution is run in this context. Contains an ordered list of tasks and can be run repeatedly and loop over the mapped hosts over each task.
+Plays can include variables as well as tasks.
+
+Roles
+.....
+
+A limited distribution of reusable ansible content (tasks, handlers, variables, plugins, templates and files) that can be referenced inside a Play.
 
-Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`.
+
+Tasks
+.....
+The units of action in Ansible, must be contained in a Play, directly or via another playbook ojbect contained in a Play (Role, or imported/included task list file).
+You can execute a single task once with an ad hoc command ``ansible`` or ``ansible-console`` (both create a virtual Play).
+
+Handlers
+........
+Tasks that only execute when notified by a previous task which resulted in a 'changed' status.
 
 Modules
 =======
 
 The units of code Ansible executes. Each module has a particular use, from administering users on a specific type of database to managing VLAN interfaces on a specific type of network device. You can invoke a single module with a task, or invoke several different modules in a playbook. Starting in Ansible 2.10, modules are grouped in collections. For an idea of how many collections Ansible includes, take a look at the :ref:`list_of_collections`.
 
-Tasks
-=====
-
-The units of action in Ansible. You can execute a single task once with an ad hoc command.
+Collections
+===========
 
-Playbooks
-=========
+Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`. Unlike roles you can use any part of a collection without importing the whole thing.
 
-Ordered lists of tasks, saved so you can run those tasks in that order repeatedly. Playbooks can include variables as well as tasks. Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
+AAP
```

Comment:
sandra mentioned how we dont typically mention AAP in the docs, i just wanted to draw attention to this